kong/plugins/session/storage/kong.lua

```diff
@@ -38,15 +38,36 @@ local function load_session_from_cache(key)
   return kong.cache:get(cache_key, nil, load_session_from_db, key)
 end
 
+local function insert_session_metadata(metadata, session)
+  if not metadata then
+    return
+  end
+
+  local audiences = metadata.audiences
+  local subjects  = metadata.subjects
+  local count     = #audiences
+  for i = 1, count do
+    kong.db.session_metadatas:insert({
+      sid      = session.session_id,
```

Comment:
According the DDL of table `session_metadata`, the `sid` is a unique column, however, the insert operation uses the same `sid` in the loop. Is it a potential bug here?

Reply:
The column `sid is` not unique in the table `session_metadata`. According to the `resty.session`. The one session allows multiple subjects and audiences.

Reply:
```sql
      CREATE TABLE IF NOT EXISTS session_metadatas(
        id            uuid,
        session_id    uuid                    REFERENCES "sessions" ("id") ON DELETE CASCADE,
        sid           text UNIQUE,
        subject       text,
----- snip ----
```

The `sid` is unique in DDL.👀

Reply:
Oh. My mistake. I will fix it. Thanks.

Reply:
By the way, is the `sid` essential here? Does it equal to `session.id`?

Reply:
It is a redundant design. With this field, the metadata data can be directly queried using `sid`.